gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenViewModel.java

```diff
@@ -80,7 +78,7 @@
             .switchMapSingle(
                 newFeature ->
                     featureRepository
-                        .saveFeature(newFeature, authManager.getCurrentUser())
+                        .saveFeature(newFeature)
```

Comment:
Why do we remove the current user here?

Reply:
`AuthenticationManager` is now moved to `FeatureRepository` for simplicity.